target/scripts/check-for-changes.sh

```diff
@@ -66,8 +66,8 @@ do
 
       for FILE in ${CHANGED}
       do
-        case ${FILE} in
-          /etc/letsencrypt/acme.json)
+        case "${FILE}" in
+          "/etc/letsencrypt/acme.json" )
```

Comment:
Currently test 111 is failing:
```
not ok 111 can detect changes
# (from function `assert_output' in file test/test_helper/bats-assert/src/assert.bash, line 231,
#  in test file test/mail_ssl_letsencrypt.bats, line 118)
#   `assert_output --partial "Change detected"' failed
# 
# -- output does not contain substring --
# substring (1 lines):
#   Change detected
# output (4 lines):
#   postfix: stopped
#   postfix: started
#   dovecot: stopped
#   dovecot: started
# --
# 
```

As this test is checking the ssl components it might be introduced with this change but not sure yet.

Reply:
Might be that only the expected output is missing now as the test output shows that services got restarted which would indicate that a change was detected.

Reply:
Seems like even with the partial match Bats cannot detect the output of

``` BASH
_notify 'inf' "${LOG_DATE} Change detected"
```

which seems odd to me. At least the partial match should do the job. Otherwise everything is fine. We're still at the problem with Bats not being able to match the `_notify` output... But this time, not even `--partial` will do, apparently.